perllib/FixMyStreet/App/Form/Wizard.pm

```diff
@@ -53,6 +53,7 @@ sub next {
         $next = $next->(
             $self->saved_data,
             $self->c->req->params,
+            $self, # Ugh
```

Comment:
I don't like this; I feel 'next' should just take the form/$self as the first (and only?) arg - but I didn't want to go around refactoring all existing 'next' subs.

Reply:
I think it's the perfect time to do that refactor, tbh. There aren't *that* many (I count 10?) and it'd make them all a bit clearer. I agree that `$self` should be the only arg. The existing test coverage should be a good safety net!

Reply:
Hmm I've got stuck on this. For some reason, when I just use $self, and then use `$self->c->req->params` in e.g. the 'next' sub for `has_page choose_date_earlier` in the Bulky.pm form, I get `Can't call method "c" on an undefined value`. Like it's not getting a form object at all, despite the original passing in of `$self->c->req->params` working fine? I don't know if I am overlooking something really silly 😞